coderd/notifications_test.go

```diff
@@ -376,3 +377,48 @@ func TestNotificationTest(t *testing.T) {
 		require.Len(t, sent, 0)
 	})
 }
+
+const (
+	// These are valid keys for a web push subscription.
+	// DO NOT REUSE THESE IN ANY REAL CODE.
+	validEndpointAuthKey   = "zqbxT6JKstKSY9JKibZLSQ=="
+	validEndpointP256dhKey = "BNNL5ZaTfK81qhXOx23+wewhigUeFb632jN6LvRWCFH1ubQr77FE/9qV1FuojuRmHP42zmf34rXgW80OvUVDgTk="
```

Comment:
could we generate them or is there a point in keeping const in the test file?

Reply:
These are the keys generated in `webpush.SendNotificationWithContext`. We'd essentially have to duplicate all of that code as it's not exposed by the package. Plus generating them isn't free either.